db/migrations/20181221164327-create-deleted-subscription.js

```diff
@@ -0,0 +1,48 @@
+module.exports = {
+  up: (queryInterface, Sequelize) => queryInterface.createTable('DeletedSubscriptions', {
+    id: {
+      allowNull: false,
+      autoIncrement: true,
+      primaryKey: true,
+      type: Sequelize.BIGINT,
+    },
```

Comment:
should we also have a column that represents the `id` from the `Subscriptions` table? Since the primary id in this new table will likely have a different value

Reply:
hmmm, what do you mean? I don't think we care about the original subscription id. But if we care I think the easiest solution would be to make the id the same and drop the `autoIncrement` in the `DeletedSubscriptions` table. What do you think?

Reply:
> hmmm, what do you mean? I don't think we care about the original subscription id.

I was thinking from the perspective of needing to reconcile a deleted subscription with perhaps a log message somewhere that includes just the subscription id (but nothing else).

> But if we care I think the easiest solution would be to make the id the same and drop the autoIncrement in the DeletedSubscriptions table. What do you think?

Agree, this seems like the better solution 👍